src/buildApplication/generateGlobalJs.ts

```diff
@@ -15,7 +15,15 @@ export function generateGlobalJs(): string {
 			env: new Proxy(
 				{},
 				{
-					get: (_, property) => Reflect.get(__ENV_ALS__.getStore(), property),
+					get: (_, property) => {
+						${/* TODO: remove try-catch ASAP (after runtime fix) @dario */ ''}
+						try {
+							const result = Reflect.get(__ENV_ALS__.getStore(), property);
+							return result;
+						} catch (e) {
+							return undefined;
```

Comment:
@GregBrimble this is kind of broken.... I don't think we can do much better (but the runtime fix should come quite soon, this or next week)

(it's either this or reverting the ALS PR I think)